Source/sourcekitten/CompleteCommand.swift

```diff
@@ -51,6 +53,13 @@ struct CompleteCommand: CommandProtocol {
         }
 
         var args = ["-c", path]
```

Comment:
I don't think we should be mixing user-provided arguments with SPM-based ones. How's this instead?

```swift
var args: [String]
if !options.spmModule.isEmpty {
    guard let module = Module(spmName: options.spmModule) else {
        return .failure(.invalidArgument(description: "Bad module name"))
    }
    args = module.compilerArguments
} else {
    args = ["-c", path] + options.compilerargs
    if args.index(of: "-sdk") == nil {
        args.append(contentsOf: ["-sdk", sdkPath()])
    }
}
```